taca/backup/backup.py

```diff
@@ -69,12 +74,16 @@ def collect_runs(self, ext=None, filter_by_ext=False):
                     elif not os.path.isdir(os.path.join(adir, item)):
                         continue
                     if re.match(filesystem.RUN_RE, item) and item not in self.runs:
-                        self.runs.append(run_vars(os.path.join(adir, item)))
+                        run_type = self._get_run_type(self.run)
+                        archive_path = self.archive_dirs[run_type]
+                        run = run_vars(os.path.join(adir, item), archive_path)
+                        if self._is_ready_to_archive(run):
+                            self.runs.append(run)
 
     def avail_disk_space(self, path, run):
         """Check the space on file system based on parent directory of the run."""
         # not able to fetch runtype use the max size as precaution, size units in GB
-        illumina_run_sizes = {'hiseq': 500, 'hiseqx': 900, 'novaseq': 1800, 'miseq': 20, 'nextseq': 250}
+        illumina_run_sizes = {'hiseq': 500, 'hiseqx': 900, 'novaseq': 1800, 'miseq': 20, 'nextseq': 250, 'NovaSeqXPlus': 3600}
```

Comment:
We should do something about this one in the future. It's a bit too conservative at the moment. Perhaps have a better resolution on what type of flowcell is used? But that's a different task.